### DISCUSSION TITLE
Show Vulnerability reports on the package homepage.

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#150] [Website]
Can a link to vulnerability report (at-least those reported from npm audit) can be added to the homepage if the package has any vulnerabilities? Knowing these information prior to installation will be useful.

**Comment by ljharb** [comment]
In general I’d agree, but this seems like it’d worsen the problem that currently exists, where the vast majority of transitive audit warnings are false positives.

**Comment by MylesBorins** [comment]
Hey @sibiraj-sr, thanks for opening this discussion!

Are you imagining that this would be for any / all known vulnerabilities for a package? Only the latest vulnerabilities? Only unpatched vulnerabilities?

**Reply by ghost** [reply]
Kind of both. Most of the times a package having a vulnerability is known only after installation from audit. It would be nicer to show the vulnerabilities for the package's latest version if not patched, in the homepage itself.

Also a tab like this or some better representation for listing all vulnerabilities of this package would be a useful addition

<img width="1324" alt="Screenshot 2021-01-15 at 11 48 55 AM" src="https://user-images.githubusercontent.com/45618743/104753707-5e07d700-577e-11eb-95bb-611c0b9e2aae.png">

**Reply by ljharb** [reply]
Are you asking for just vulns for ngx-editor to show up there, or also for vulns of ngx-editor's dependency graph?

**Reply by ghost** [reply]
That tab is for that package alone. The second I mentioned was just for convenience. Sometimes it is hard to find the advisory of the package. It would be nice if they are listed at one place which is alongside with the packages page.

**Reply by ghost** [reply]
May be it could be named as `Advisories` instead of `Vulnerabilities`
